internal/method/reference_processor.go

```diff
@@ -169,12 +179,13 @@ func getTypeCodeFromPath(path string, nameSuffix ...string) *jen.Statement {
 	return jen.Op("&").Qual(pkg, name).Values()
 }
 
-func getFuncCodeFromPath(path string) *jen.Statement {
-	words := strings.Split(path, ".")
-	if len(words) == 1 {
-		return jen.Id(path)
+func getFuncCodeFromPath(path string) (*jen.Statement, error) {
+	parts := regexFunctionCall.FindStringSubmatch(path)
+	if len(parts) != 4 {
```

Comment:
A comment showing example values for each part and explaining why `4` is the right number here would be great.

Reply:
Done.